package.json

```diff
@@ -20,11 +24,18 @@
   },
   "homepage": "https://topggjs.rtfd.io",
   "devDependencies": {
+    "@top-gg/eslint-config": "^0.1.8",
```

Comment:
```suggestion
```
This dependency does not exist or haven't been published.

Reply:
https://github.com/top-gg/eslint-config

Reply:
Deleting the eslint config dependency is obviously not the correct way to resolve this issue